src/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+  font-weight: 300;
+}
+
+.container {
+  position: relative;
+  box-sizing: border-box;
+}
```

Comment:
if you need this container just for the sake of positioning image, then delete it and place the image inside form